src/plugins/kibana/public/settings/sections/indices/directives/kbn_settings_indices.html

```diff
@@ -0,0 +1,38 @@
+<div class="col-md-2 sidebar-container">
+  <div class="sidebar-list">
+    <div class="sidebar-list-header">
+      <h5>
+        Index Patterns&nbsp;
+        <a
+          ng-if="edittingId"
```

Comment:
Looks like it :)   I just moved this existing code from index.js into its own file, I think I'll leave it alone to avoid accidentally breaking anything.